templates/repo/diff/blob_excerpt.tmpl

```diff
@@ -3,36 +3,38 @@
 	<tr class="{{.GetHTMLDiffLineType}}-code nl-{{$k}} ol-{{$k}}">
 		{{if eq .GetType 4}}
 			<td class="lines-num lines-num-old" data-line-num="{{if $line.LeftIdx}}{{$line.LeftIdx}}{{end}}">
+				<div class="gt-df">
 				{{if or (eq $line.GetExpandDirection 3) (eq $line.GetExpandDirection 5)}}
-					<a role="button" class="blob-excerpt" data-url="{{$.RepoLink}}/blob_excerpt/{{PathEscape $.AfterCommitID}}" data-query="{{$line.GetBlobExcerptQuery}}&style=split&direction=down&wiki={{$.PageIsWiki}}" data-anchor="{{$.Anchor}}">
+					<button class="code-expander-button gt-f1" data-url="{{$.RepoLink}}/blob_excerpt/{{PathEscape $.AfterCommitID}}" data-query="{{$line.GetBlobExcerptQuery}}&style=split&direction=down&wiki={{$.PageIsWiki}}" data-anchor="{{$.Anchor}}">
```

Comment:
It looks like these styles could be put on `<td class="lines-num` and `<button class="code-expander-button`, then no need these `gt-` helpers.

Reply:
Yes, I don't quite like it that some of these button have the helper, while some don't.

Reply:
Who hasn't?
I thought I got them all by now…

Reply:
Currently all have it. Before, there were some without in `templates/repo/diff/blob_excerpt.tmpl`.

Reply:
But why not put the "flex" styles in CSS class `lines-num` and `code-expander-button` ?

Reply:
That doesn't work as it changes the layout: 
![image](https://github.com/go-gitea/gitea/assets/51889757/16cf1ad1-f240-47d5-b298-01040b8d6095)
That's not how these buttons are supposed to be centered.

Reply:
Probably because table cells don't really suport flexbox, so wrapper may be required, but I would probably put one around all of these buttons, not only some of them.

Reply:
> Currently all have it